src/image_transfer.py

```diff
@@ -251,6 +257,9 @@ def main():
             LOGGER.info(f"Transferred image {autopatch_image}")
         else:
             LOGGER.info(f"Image {autopatch_image} cannot be transferred.")
+            raise ImageCannotBeTransferredException(
+                f"Image {autopatch_image} could not be transferred."
+            )
```

Comment:
Is there a reason to have both an `INFO` level message as well as the Exception with its own `ERROR` level log message?

Reply:
Thanks, I have removed the logging statement